examples/kubernetes/README.md

```diff
@@ -21,7 +21,7 @@ You'll need [Go 1.3+](http://golang.org/doc/install) in order to build the
 ### Build and install vtctlclient
 
 ```
-$ go get github.com/youtube/vitess/go/cmd/vtctlclient
+vitess/examples/kubernetes$ go build ../../go/cmd/vtctlclient
```

Comment:
I thought it would but I still get that error.  Removing influxdb plugin from vtctlclient at least allows for running go build in the vtctlclient folder.